back-office/catalog/products/catalog-products-add-edit.md

```diff
@@ -155,7 +153,9 @@ The image is open in its default size. You can close it by clicking on the cross
   * The cross to close the frame and unselect images
   * Caption field: You can enter a caption which will be applied to all images. If some selected images already had captions, they are replaced when clicking on Save.
 
-    If more than one image is selected the checkbox "cover" disappears
+    If more than one image is selected the checkbox "cover" disappears.
+    
+The **Features section** is displayed if the Feature option in the Advanced Parameters / Performance / Optional features page enables otherwise the section is hidden.
```

Comment:
```suggestion
The **Features section** is displayed if the Feature option in Advanced Parameters > Performance > Optional features page is enabled. Otherwise the section is hidden.
```